packages/acs-ui-javascript-loaders/package.json

```diff
@@ -0,0 +1,72 @@
+{
+    "name": "@internal/acs-ui-javascript-loaders",
+    "version": "1.18.0-beta.1",
+    "description": "Component bindings specific to components that integrate with the Azure Communication Services Calling JavaScript SDK",
+    "module": "dist/dist-esm/index.js",
+    "main": "dist/dist-cjs/index.js",
+    "types": "dist/dist-esm/index.d.ts",
+    "scripts": {
+      "build": "rushx _by-flavor \"rushx _build:by-flavor\"",
+      "build:esm": "npm run _if-preprocess && rushx preprocess && tsc -project tsconfig.preprocess.json || (if-env COMMUNICATION_REACT_FLAVOR=beta && tsc)",
+      "build:cjs": "rollup -c  --failAfterWarnings",
+      "build:watch": "rushx build",
+      "clean": "rimraf dist && rimraf preprocessed",
+      "preprocess": "cpx \"./src/**\" ./preprocessed && babel ./src --out-dir ../preprocessed --extensions \".ts,.tsx\" --keep-file-extension --config-file ./.babelrc.js --relative && rimraf ../preprocessed",
+      "prettier": "prettier --no-error-on-unmatched-pattern --write --config ../../.prettierrc --ignore-path=../../.prettierignore \"**/*.js\" \"**/*.jsx\" \"**/*.ts\" \"**/*.tsx\"",
+      "prettier:check": "prettier --no-error-on-unmatched-pattern --check --config ../../.prettierrc --ignore-path=../../.prettierignore \"**/*.js\" \"**/*.jsx\" \"**/*.ts\" \"**/*.tsx\"",
+      "test": "rushx _by-flavor \"rushx _test:by-flavor\"",
+      "test:ci-coverage": "rushx _by-flavor \"rushx _test:by-flavor --coverage\"",
+      "test:coverage": "rushx _by-flavor \"rushx _test:by-flavor --coverage --coverageReporters=lcov\"",
+      "lint": "eslint --max-warnings 0 \"*/**/*.{ts,tsx}\"",
+      "lint:fix": "rushx lint --fix --",
+      "lint:quiet": "rushx lint -- --quiet",
+      "_if-preprocess": "if-env COMMUNICATION_REACT_FLAVOR=stable || if-env COMMUNICATION_REACT_FLAVOR=beta-release",
+      "_if-beta": "if-env COMMUNICATION_REACT_FLAVOR=beta || if-env COMMUNICATION_REACT_FLAVOR=beta-release",
+      "_build:by-flavor": "rushx clean && rushx build:esm",
+      "_by-flavor": "rushx _current-flavor && env-cmd -f ../../common/config/env/.env --use-shell",
+      "_current-flavor": "echo You are running under COMMUNICATION_REACT_FLAVOR: && env-cmd -f ../../common/config/env/.env node -p process.env.COMMUNICATION_REACT_FLAVOR",
+      "_test:by-flavor": "(npm run _if-preprocess && rushx preprocess || if-env COMMUNICATION_REACT_FLAVOR=beta && echo \"skip preprocess\") && jest"
+    },
+    "author": "",
+    "license": "ISC",
+    "dependencies": {
+      "@azure/communication-common": "^2.3.1",
+      "@internal/acs-ui-common": "1.18.0-beta.1",
+      "@internal/react-composites": "1.18.0-beta.1",
+      "@fluentui/react": "^8.117.1",
+      "@azure/logger": "^1.0.4"
+    },
+    "peerDependencies": {
+      "react": ">=16.8.0 <19.0.0",
```

Comment:
We have a check for react >= 18 in the code. Should this also be the case here?

Reply:
in order for Rush to not complain about mismatched deps yes we still need to have this